src/registrationtools/spatial_registration.py

```diff
@@ -553,11 +562,12 @@ def compute_trsfs(self, p: trsf_parameters):
                     + res_trsf
                     +  # ' -res-voxel-trsf ' + res_voxel_trsf + \
                     # ' -res ' + flo_out +\
-                    " -composition-with-initial",
+                    #" -composition-with-initial"
+                    th, 
```

Comment:
Is there a reason why you remove the composition with the initial transformation? It does not seem correct but I might be wrong.